app/src/main/res/values/styles.xml

```diff
@@ -563,9 +563,24 @@
         <item name="tint">?attr/primary_color</item>
     </style>
 
+    <style name="DialogIconWithSecondaryTint" parent="MaterialAlertDialog.Material3.Title.Icon.CenterStacked">
+        <item name="tint">?attr/secondary_color</item>
+    </style>
+
+    <style name="AlertDialogTheme.IconWithSecondaryTint" parent="AlertDialogTheme.Icon">
+        <item name="materialAlertDialogTitleIconStyle">@style/DialogIconWithSecondaryTint</item>
+    </style>
+
     <style name="AlertDialogTheme.Icon">
         <item name="materialAlertDialogTitlePanelStyle">@style/MaterialAlertDialog.Material3.Title.Panel.CenterStacked</item>
         <item name="materialAlertDialogTitleIconStyle">@style/DialogIconWithTint</item>
+        <item name="materialAlertDialogTitleTextStyle">@style/centeredTitleText</item>
+    </style>
+
+    <style name="centeredTitleText" parent="MaterialAlertDialog.Material3.Title.Text.CenterStacked">
```

Comment:
Have you confirmed with the designer about the global change?

Also, can we make the name more dialog-specific? something like: `AlertDialogTitleTextStyle.Centered`

Reply:
I have not confirmed with design but we are making the title centered for all dialogs. I have asked design for confirmation.

Reply:
I created a style AlertDialogTheme.Icon.CenteredText, so that we can have variation in style. If we want title centered we can use this one otherwise the other one.